content/applications/productivity/discuss/overview/get_started.rst

```diff
@@ -76,12 +83,12 @@ Mentions in the chat and on the Chatter
    task’s followers, and secondly on *Employees*. If the record being searched does not match with
    either a follower or employee, the scope of the search becomes all partners.
 
-Chat status
+User status
 -----------
 
 It is helpful to see what colleagues are up to and how quickly they can respond to messages by
-checking their *Status*. The status is shown on the left side of a contact’s names on Discuss’
-sidebar and on the *Messaging menu*.
+checking their *status*. The status is shown on the left side of a contact’s name on the *Discuss*
+sidebar, on the *Messaging menu* and when listed in the *chatter*.
```

Comment:
Apostrophe fix, can use guilabel on Discuss since it is displayed on the UI in this instruction, lowercase messaging menu
```suggestion
checking their *status*. The status is shown on the left side of a contact's name on the
:guilabel:`Discuss` sidebar, on the *messaging menu* and when listed in the *chatter*.
```